nuxt.config.js

```diff
@@ -266,6 +266,24 @@ export default defineNuxtConfig({
     }, // https://github.com/nuxt-community/apollo-module#options
   },
 
+  babel: {
+    presets() {
+      return [
+        [
+          '@nuxt/babel-preset-app',
+          {
+            overrides: [
+              {
+                test: './vendor/large.min.js',
+                compact: true,
+              },
+            ],
+          },
+        ],
+      ]
+    },
+  },
```

Comment:
What is the purpose of this? only put model-viewer in transpile doesn't work?

Reply:
it is there to silence a rather unimportant warning by babel https://github.com/babel/babel/discussions/13676

i just pushed a much more simple config and added a comment regarding its purpose